geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/Dockerfile_tmpl

```diff
@@ -32,13 +32,11 @@ RUN true
 COPY --from=builder /usr/lib/node_modules/ngeo/dist/* /etc/static-ngeo/
 COPY --from=builder /etc/static-ngeo/* /etc/static-ngeo/
 COPY --from=builder /app/alembic.ini /app/alembic.yaml ./
-RUN chmod go+w /etc/static-ngeo/ \
-    /app/{{package}}_geoportal/locale/ \
-    /app/{{package}}_geoportal/locale/*/LC_MESSAGES/{{package}}_geoportal-client.po
+RUN chmod go+w /etc/static-ngeo/
```

Comment:
Should  be added to the docker ignore...

Reply:
Good catch